polyphemus/lib/commands.rb

```diff
@@ -268,7 +269,7 @@ def execute
           # Magma does not support delete right now, so we restrict the models, but we can unlink and
           # delete the files in metis
           restrict!(patient, delete_metis_files: true)
-        elsif should_be_restricted && ! patient['restricted']
```

Comment:
So now we query every patient, and based on their consent status start the file moving proces every time.